src/libs/actions/Subscription.ts

```diff
@@ -279,6 +292,44 @@ function clearOutstandingBalance() {
     API.write(WRITE_COMMANDS.CLEAR_OUTSTANDING_BALANCE, null, onyxData);
 }
 
+function cancelBillingSubscription(cancellationReason: FeedbackSurveyOptionID, cancellationNote: string) {
+    const successData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.NVP_PRIVATE_CANCELLATION_DETAILS,
+            value: [
+                ...cancellationDetails,
```

Comment:
For both success and failure we're setting `...cancellationDetails`?

Reply:
Yes, this onyx key is an array, it contains a history of previous cancellations, it has to remain unchanged. Hope that answers your question!